2023/dotnetvale/data.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
```

Comment:
We need the GitHub handles of anyone who is allowed to modify your community folder "dotnetvale", including yours!

Reply:
Ok, i really forgot this

Reply:
Something is wrong. I cannot see any changes!